scylladb/sphinx-scylladb-theme
# Update to Sphinx 7.0 & enable parallell builds

Related issue https://github.com/scylladb/sphinx-scylladb-theme/issues/887 and https://github.com/scylladb/sphinx-scylladb-theme/issues/860

- [x] Update to Sphinx 7
- [x] Update min Python 3.9
- [x] Enable parallel builds
- [x] Fix RemovedInSphinx80Warning: Sphinx 8 will drop support for representing paths as strings. Use "pathlib.Path" or "os.fspath" instead warnings.
- [x] backport https://github.com/scylladb/sphinx-scylladb-theme/pull/890 
- [x] Bump multiversion version
- [x] Test against sphinx-scylladb-theme repo
- [x] Test against scylla opensource repo

## How to test

1. Build the docs.
2. Check that ther are not errors.
3. Check that the footer shows Sphinx 7.0.x
